components/org.wso2.carbon.identity.authz.valve/src/main/java/org/wso2/carbon/identity/authz/valve/util/Utils.java

```diff
@@ -50,6 +52,22 @@ public static String getTenantDomainFromURLMapping(Request request) {
         return domain;
     }
 
+    public static String getOrganizationIdFromURLMapping(Request request) {
+
+        String requestURI = request.getRequestURI();
+        String organizationId = StringUtils.EMPTY;
+
+        if (requestURI.contains("/o/")) {
+            String temp = requestURI.substring(requestURI.indexOf("/o/") + 3);
```

Comment:
Can we assure about the + 3 value?

Reply:
yes. Here 3 means the length of the string `/o/`